plugins/woocommerce-admin/client/marketing/data/actions.js

```diff
@@ -39,6 +39,16 @@ export function receiveRecommendedPlugins( plugins, category ) {
 	};
 }
 
+export function removeRecommendedPlugin( pluginSlug, category ) {
```

Comment:
In case we finally hold this action here it would be nice to add JSDOC comment at the top

Reply:
Done in [`aac6c8b` (#35542)](https://github.com/woocommerce/woocommerce/pull/35542/commits/aac6c8bffd88ac6f07f752d6ee49eaa5e1a159b7).